sound/soc/sof/ipc4-topology.c

```diff
@@ -1384,7 +1384,7 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 	int *ipc_config_size;
 	u32 **data;
 	int ipc_size, ret;
-	u32 out_ref_rate, out_ref_channels, out_ref_valid_bits;
+	u32 out_ref_rate = 0, out_ref_channels = 0, out_ref_valid_bits = 0;
```

Comment:
These should be in separate lines as they are initialized.

Reply:
Ack, updated in v2, thanks!